app/qml/main.qml

```diff
@@ -268,7 +268,7 @@ ApplicationWindow {
       }
       else if ( locationPermission.status === Qt.Denied ) {
         __inputUtils.log("Permissions", "Location permission is denied")
-        showMessage( qsTr( "Location permission is required to show your location on map. Please enable it in system settings." ) );
+        __notificationModel.addInfo( qsTr( "Location permission is required to show your location on map. Please enable it in system settings." ) );
```

Comment:
I wonder if the text here is not too long for the notification. Can you check @iiLubos please?

Reply:
It looks fine, notification text can be shown in 3 rows.
https://github.com/MerginMaps/mobile/assets/72412563/79fdbf8d-26e4-44fe-9e25-afa7e0e22af6